optuna/storages/_rdb/models.py

```diff
@@ -522,6 +572,58 @@ def all(cls, session: orm.Session) -> List["TrialValueModel"]:
         return session.query(cls).all()
 
 
+class TrialIntermediateValueModel(BaseModel):
+    __tablename__ = "trial_intermediate_values"
+    __table_args__: Any = (UniqueConstraint("trial_id", "step"),)
+    trial_intermediate_values_id = Column(Integer, primary_key=True)
+    trial_id = Column(Integer, ForeignKey("trials.trial_id"))
+    step = Column(Integer)
+    value = Column(Float)
```

Comment:
How about naming this "intermediate_value"? Seems more consistent (maybe not with the higher layers such as `Trial.report`, but looking at the schema definitions).

Reply:
Seems like this comment is unaddressed or did you meant that we shouldn't rename it "intermediate_value"?

Reply:
Ops! I misunderstood that we should `trial_intermediate_values_id` -> `trial_intermediate_value_id`. I will change `value` -> `intermediate_value` now. Thanks for pointing.